docs/source/sctool/partials/sctool_backup_list.yaml

```
@@ -45,11 +45,11 @@ options:
     Specifies maximal snapshot `date` expressed in RFC3339 form or 'now[+duration]', ex. 'now+3d2h10m'.
     Valid units are:
 
-    'd' - days
-    'h' - hours
-    'm' - minutes
-    's' - seconds
-    'now' - happens immediately
+    * ``d`` - days
+    * ``h`` - hours
+    * ``m`` - minutes
+    * ``s`` - seconds
+    * ``now`` - happens immediately
 - name: min-date
   usage: |
     Specifies minimal snapshot `date` expressed in RFC3339 form or 'now[+duration]', ex. 'now+3d2h10m'.
```

```suggestion
    Specifies minimal snapshot ``date`` expressed in RFC3339 form or ``now[+duration]``, ex. ``now+3d2h10m``.
Write
```

@mmatczuk Could you please update the code in ``master`` to generate this output (applies to all inline code blocks). The current documentation is using three different forms (', `` and `).